src/Client/Web/Playground/Bit.Client.Web.BlazorUI.Playground/Web/wwwroot/scripts/app.ts

```diff
@@ -1,5 +1,9 @@
-﻿function copyCodeSample(element){
-    element.select();
-    document.execCommand('copy');
-    element.setSelectionRange(0, 0);
+﻿function toggleBodyOverflow(isMenuOpen: boolean) {
+    if (window.innerWidth < 1024) {
+        if (isMenuOpen) {
+            document.body.style.overflow = "hidden";
+        } else {
+            document.body.style.overflow = "auto";
+        }
+    }
```

Comment:
plz move this file from `wwwroot` to a specific folder in the root of the application, just like the component project.